src/commands/project/create-variable-group.test.ts

```diff
@@ -297,37 +204,19 @@ describe("setVariableGroupInBedrockFile", () => {
 
   test("Should pass adding a valid variable group name when bedrock file exists with empty variableGroups", async () => {
     // Create random directory to initialize
-    const randomTmpDir = path.join(os.tmpdir(), uuid());
-    fs.mkdirSync(randomTmpDir);
-
-    // create bedrock file to simulate the the use case that `spk project init` ran before
-    const bedrockFileData: IBedrockFile = {
-      rings: {},
-      services: {},
-      variableGroups: []
-    };
-
-    const asYaml = yaml.safeDump(bedrockFileData, {
-      lineWidth: Number.MAX_SAFE_INTEGER
-    });
-
-    fs.writeFileSync(path.join(randomTmpDir, "bedrock.yaml"), asYaml);
+    const randomTmpDir = createBedrockYaml();
```

Comment:
does file gets created in randomTempDir? or Default dir?

Reply:
in random temp dir.